### ISSUE TITLE
WordPress iOS App can't upload to YunoHost WordPress installation

### ISSUE THREAD

**Issue by KevinGimbel** [YunoHost-Apps/wordpress_ynh#204]
### Describe the bug

WordPress iOS App does not work with a fresh installation of WordPress 6.1 from YunoHost.
Other self-hosted WordPress instances work fine.

### Context

- Hardware: CPX31 instance from Hetzner (4CPU, 8GB)
- YunoHost version: 11.0.10.2 (stable)
- I have access to my server: Through SSH
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch: Latest released YunoHost WordPress package (
- If upgrading, current package version: *can be found in the admin, or with `yunohost app info $app_id`*

### Steps to reproduce

Install YunoHost
Install WordPress
Install iOS WordPress App
Connect WordPress to App
Try and upload anything (Post, media, page,...)

### Expected behavior

I expect to be able to use the app with my WordPress installation

### Logs

No specific logs are available as far as I can see.

**Comment by walterebert** [comment]
Uploading images with the app works for me with Yunohost 11.1.0.2 . Did you try to install updates from the WordPress admin?

**Comment by KevinGimbel** [comment]
There are no updates to install at the moment, but the web admin works fine - I can edited posts, create pages, install and remove plugins, ... - only the WordPress App can't connect to my server. 

I've checked the DNS settings as well because I use multiple apps and different domains on the same server, but they seem to be right for all I wanna use. I configured everything except for e-mail as I use an external e-mail service.

I do use an "floating IP", so YunoHost reports that the IP is wrong but I guess I can ignore that? 

---

Maybe related, but on the same server I have a  Mastodon instance which cannot find the WordPress instance which uses a activitypub plugin. From mastodon.social I can search for "@kevin@kevingimbel.de" and find my wp site, interact with it, and follow it. 
When I do the same from my YunoHost installed Mastodon I get a `HostValidationError` error.
